roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -19,7 +19,8 @@ public_workers = {{ keystone.public_workers }}
 connection=mysql://keystone:{{ secrets.db_password }}@{{ endpoints.db }}/keystone?charset=utf8
 
 [identity]
-driver = keystone.identity.backends.sql.Identity
```

Comment:
I think we can leave this in here, as that means any undefined domain will use sql Identity, such as the Default domain, or any domain a customer creates.